src/__swaps__/screens/Swap/components/UserAssetsSync.tsx

```diff
@@ -29,8 +29,10 @@ export const UserAssetsSync = () => {
         data.forEach(asset => {
           if (filter === 'all' || asset.chainId === filter) {
             if (searchQuery) {
-              const stringToSearch = `${asset.name} ${asset.symbol} ${asset.address}`.toLowerCase();
-              if (stringToSearch.includes(searchQuery)) {
+              const nameMatch = asset.name.toLowerCase().includes(searchQuery);
+              const symbolMatch = asset.symbol.toLowerCase().startsWith(searchQuery);
+              const addressMatch = asset.address.toLowerCase().startsWith(searchQuery);
+              if (nameMatch || symbolMatch || addressMatch) {
```

Comment:
why the change here?

Reply:
oh hmm i see nvm, good catch

Reply:
we can prob improve later also. for example, rank `startsWith` matches for name higher than `includes` matches for name, if that makes sense.